test/new_relic/recipes/helpers/send_deployment_test.rb

```diff
@@ -0,0 +1,59 @@
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+# frozen_string_literal: true
+
+require_relative '../../../test_helper'
+require_relative '../../../../lib/new_relic/recipes/helpers/send_deployment'
+
+module Newrelic
```

Comment:
We use a capital "R" for "Relic" in most situations, however, not in all situations. It probably doesn't matter here.

Reply:
addressed with [bcebb46](https://github.com/newrelic/newrelic-ruby-agent/pull/1654/commits/bcebb464c4274404fe3433032555cac283da0755)